libs/openFrameworksCompiled/project/msys2/config.msys2.default.mk

```diff
@@ -236,6 +236,7 @@ PLATFORM_CORE_EXCLUSIONS += $(OF_LIBS_PATH)/utf8/%
 PLATFORM_LIBRARIES += ksuser opengl32 gdi32 msimg32 glu32 dsound winmm strmiids #dxguid
 PLATFORM_LIBRARIES += uuid ole32 oleaut32 setupapi wsock32 ws2_32 Iphlpapi Comdlg32
 PLATFORM_LIBRARIES += freeimage boost_filesystem-mt boost_system-mt
+PLATFORM_LIBRARIES += mfreadwrite xaudio2
```

Comment:
Yeah I added some here, but maybe there needs to be more

Reply:
Going to add the remaining mf libs for msys2, since the video player is not included in msys2 at the moment